docs/cli.md

```diff
@@ -1,178 +1,59 @@
 
-With the defaults described above, `.js` files are loaded via `require()`, whereas `.mjs` files are loaded via `import()`.
-
-Alternatively, you can use either or both of these options to explicitly load support files before executing the features:
+```json
+{
+  "scripts": {
+    "cucumber": "cucumber-js"
+  }
+}
+```
 
-- `--require <GLOB|DIR|FILE>` - loads via `require()` (legacy)
-- `--import <GLOB|DIR|FILE>` - loads via `import()`
+Or via [npx](https://docs.npmjs.com/cli/v8/commands/npx):
 
-Both options use [glob](https://github.com/isaacs/node-glob) patterns and may be used multiple times in order to e.g. load files from several different locations.
+``` shell
+$ npx cucumber-js
+```
 
-_Note that once you specify any `--require` or `--import` options, the defaults described above are no longer applied._
+**Note on global installs:** Cucumber does not work when installed globally because `@cucumber/cucumber`
+needs to be required in your support files and globally installed modules cannot be required.
 
-## Formats
+## Options
 
-Use `--format <TYPE[:PATH]>` to specify the format of the output.
+All the [standard configuration options](./configuration.md#options) can be provided via the CLI.
 
-See [Formatters](./formatters.md).
+Additionally, there are a few options that are specific to the CLI:
 
-### Officially-supported standalone formatters
+| Option             | Type       | Repeatable | Description                                                                             |
+|--------------------|------------|------------|-----------------------------------------------------------------------------------------|
+| `--config`, `-c`   | `string`   | No         | Path to your configuration file - see [Files](./configuration.md#files)                 |
+| `--profile`, `-p`  | `string[]` | Yes        | Profiles from which to include configuration - see [Profiles](./profiles.md)            |
+| `--version`, `-v`  | `boolean`  | No         | Print the currently installed version of Cucumber, then exit immediately                |
+| `--i18n-keywords`  | `string`   | No         | Print the Gherkin keywords for the given ISO-639-1 language code, then exit immediately |
+| `--i18n-languages` | `boolean`  | No         | Print the supported languages for Gherkin, then exit immediately                        |
 
-* [@cucumber/pretty-formatter](https://www.npmjs.com/package/@cucumber/pretty-formatter) - prints the feature with inline results,  colours and custom themes.
+To see the available options for your installed version, run:
 
-### Format Options
+```shell
+$ cucumber-js --help
+```
 
-You can pass in format options with `--format-options <JSON>`.
+## Exiting
 
-See [Formatters](./formatters.md).
+By default, cucumber exits when the event loop drains. Use the `exit` configuration option in order to force shutdown of the event loop when the test run has finished:
```

Comment:
Not related to the current PR: maybe we could rename that `exit` option to something else that self-explained that this is kinda `workaround`, like `forceExit`, or `forceShutdown` which is even more cryptic, so users may understand what they are doing when using it because they would have actually read the doc?

Reply:
Yeah this is a good point - perhaps we change to `forceExit` (file) and `--force-exit` and maintain `--exit` for backwards compatibility?

Reply:
Good idea to keep it for backward compatibility, but not mentioning it anymore in any documentation?
Is it possible with commander to do something like this: `--[force-]exit`?